playbooks/scenario18_create_cce.yaml

```diff
@@ -0,0 +1,111 @@
+---
+- name: Scenario 18 - CCE Lifecycle
+  hosts: localhost
+  vars:
+    prefix: scenario18-
+  tasks:
+
+    - name: Set facts
+      set_fact:
```

Comment:
please also add random part to the name as in other scenarios

Reply:
I will change this after everything is running properly.